File: CS/DisableVisualStylesModule.Win/DisableVisualStylesModuleWindowsFormsModule.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Win.SystemModule;

namespace DisableVisualStylesModule.Win {
    public sealed class DisableVisualStylesModuleWindowsFormsModule : ModuleBase {
        public DisableVisualStylesModuleWindowsFormsModule() {
            RequiredModuleTypes.Add(typeof(SystemWindowsFormsModule));
        }
    }
}

File: VB/DisableVisualStylesModule.Win/DisableVisualStylesModuleWindowsFormsModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Win.SystemModule

Namespace DisableVisualStylesModule.Win
	Public NotInheritable Class DisableVisualStylesModuleWindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			RequiredModuleTypes.Add(GetType(SystemWindowsFormsModule))
		End Sub
	End Class
End Namespace